lib/WIFI/devWIFI.cpp

```diff
@@ -262,8 +270,18 @@ static void WebUpdateForget(AsyncWebServerRequest *request)
   config.SetSSID("");
   config.SetPassword("");
   config.Commit();
-  String msg = String("Home network forgotten, please connect to access point '") + ssid + "' with password '" + password + "'";
-  sendResponse(request, msg, WIFI_AP);
+  // If we have a flashed wifi network then let's try reconnecting to that otherwise start an access point
+  if (home_wifi_ssid[0]!=0) {
```

Comment:
Might be nice to get some whitespace here... i.e. `if (home_wifi_ssid[0] != 0) {`

And also define what `0` means.... death to magic numbers!

Reply:
Also, isn't `home_wifi_ssid` the saved SSID? Are we just looking for a zero in the first char?
I'm guessing the bytes are ASCII text, which means if someone's SSID started with `0` we wouldn't accidentally fall over here?